<launch>
	<arg name="pkg_name" default="fleet_planning" doc="name of the package"/>
	<arg name="node_name" default="virtual_duckiebot_node" doc="name of the node"/>
	    <!-- Local -->
	    <node pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true"></node>

</launch>